templates/circle.yml.ejs

```diff
@@ -26,14 +26,6 @@ jobs:
       - run:
           name: Testing
           command: <%- yarn ? 'yarn' : 'npm' %> test
-  node-8:
```

Comment:
node-8 and node-10 are referred to later on (se line 61/62) so you'll need to remove those.